CodingTracker.Speedierone/CodingTracker.Speedierone/UserInput.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Data.Sqlite;
+namespace CodingTracker;
+
+internal class UserInput
+{
+    private static void ViewRecords()
+    {
+
+    }
+
+    internal static void AddRecord()
+    {
+        Console.Clear();
+        string date = Helpers.GetDate();
+        string timeStart = Helpers.GetTime();
+        string timeEnd = Helpers.GetEndTime();
+        string timeSpan = Helpers.GetTimeDifference();
```

Comment:
At the moment the code doesn't compile. 

![image](https://user-images.githubusercontent.com/108711476/231961246-76c5cdb9-6f70-434d-be91-065f7c71d55b.png)